core/src/main/scala/kafka/server/metadata/KRaftMetadataCache.scala

```diff
@@ -151,9 +151,7 @@ class KRaftMetadataCache(
    * @param topicName                   The name of the topic.
    * @param listenerName                The listener name.
    * @param startIndex                  The smallest index of the partitions to be included in the result.
-   * @param upperIndex                  The upper limit of the index of the partitions to be included in the result.
```

Comment:
There's a similar issue a few lines below with `firstTopicPartitionStartIndex` in the `getTopicMetadataForDescribeTopicResponse()` method. Can we fix that too?

Reply:
Sure, I will update the patch soon 😸